azure-documentdb-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/documentdb/DocumentDBAutoConfiguration.java

```diff
@@ -30,6 +30,7 @@
 @EnableConfigurationProperties(DocumentDBProperties.class)
 public class DocumentDBAutoConfiguration {
     private static final Logger LOG = LoggerFactory.getLogger(DocumentDBAutoConfiguration.class);
+    private static final String UserAgentSuffix = "azure-documentdb-spring-boot-starter/0.1.3";
```

Comment:
constant name check not applying to private field. still update it to upper case for better looking